app/app-controllers/doubleIconsController.js

```diff
@@ -3,41 +3,40 @@ import Astro from 'astro/astro-full';
 import BackboneEvents from 'vendor/backbone-events';
 import DoubleIconsPlugin from '../app-plugins/doubleIconsPlugin';
 
-var DoubleIconsController = function(headerId, generateLeftIcon, generateRightIcon) {
+const DoubleIconsController = function(headerId, generateLeftIcon, generateRightIcon) {
     this.headerId = headerId;
     this.generateLeftIcon = generateLeftIcon;
     this.generateRightIcon = generateRightIcon;
 };
 
-var _setLeftIcon = function(doubleIcons, address) {
+const _setLeftIcon = function(doubleIcons, address) {
     doubleIcons.setLeftIcon(address);
```

Comment:
Should this function be an `async` function and this call be `await doubleIcons.setLeftIcon(address);` ?